tests/conftest.py

```diff
@@ -11,3 +12,8 @@ def random():
     """
     with seed_random():
         yield
+
+
+@pytest.fixture(params=[False, True] if HABEMUS_TENSOR_EXT else [])
+def use_tensor_extension(request):
+    return request.param
```

Comment:
i think this will skip all the tests that use this fixture if the tensorpandas external isn't available
```suggestion
@pytest.fixture(params=[False, True] if HABEMUS_TENSOR_EXT else [False])
def use_tensor_extension(request):
    return request.param
```

Reply:
Yes you are right 👍